pom.xml

```
@@ -21,6 +21,7 @@
     <module>packages/java/engine-runtime</module>
     <module>packages/java/maven-plugin</module>
     <module>packages/java/gradle-plugin</module>
+    <module>packages/java/hilla-plugin-base</module>
```

```suggestion
    <module>packages/java/plugin-base</module>
```

I would simply name it `plugin-base`, this would look more like other technical artifact names (`generator-...`, `parser-...`) and less like something for the actual users.

---

On the other hand, maybe I'm wrong, sticking to a common pattern with, e. g., `flow-plugin-base`, is good too.

---

Well, in the Hilla project we decided not to use `hilla` in module names, if I recall correctly.

---

Oh, my bad! ;)